doc/manual/toc.tex

```diff
@@ -1,7 +1,10 @@
 \begingroup
-\fontfamily{ptm}
-\normalsize
-\fontseries{c}\selectfont
+%\fontfamily{phv}
+%\normalsize
+%\fontseries{c}\selectfont
 \setlength{\parskip}{0.05\baselineskip}
+\makeatletter
+\renewcommand*\l@section{\@dottedtocline{2}{1.3em}{2.6em}}
```

Comment:
This is obscure and looks like a kludge. Is there no better way to accomplish this?

Reply:
A better alternative would be to use the package tocloft.
Can I introduce it?

Reply:
Tocloft is part of the texlive-latex-extra package which we install in the provisioning scripts. Go for it.